services/QuillLMS/client/app/bundles/Grammar/components/grammarActivities/container.tsx

```diff
@@ -335,10 +339,26 @@ export class PlayGrammarContainer extends React.Component<PlayGrammarContainerPr
       dispatch(goToNextQuestion())
     }
 
+    getCurrentQuestion = (showTranslation) => {
+      const { session } = this.props
+      if(!session?.currentQuestion) { return null }
```

Comment:
This is purely stylistic, but on the back-end we have a linter rule that early return calls need an empty line after them to make it more obvious at a glance that it interrupts the logic of the function.  I don't know if we also want to do that on the front-end, but I figured I'd mention it in case we liked the idea.

Reply:
I like that linter rule too.

Reply:
I'll chat with Emilia about it once she's back in!